SeleniumCookies/wrapper.py

```diff
@@ -356,27 +364,30 @@ def chrome(cookie_file=None, domain_name=""):
     """Returns a cookiejar of the cookies used by Chrome. Optionally pass in a
     domain name to only load cookies from the specified domain
     """
+    logging.info('Loading cookie from Chrome')
     return Chrome(cookie_file, domain_name).load()
 
 
 def firefox(cookie_file=None, domain_name=""):
     """Returns a cookiejar of the cookies and sessions used by Firefox. Optionally
     pass in a domain name to only load cookies from the specified domain
     """
+    logging.info('Loading cookie from firefox')
     return Firefox(cookie_file, domain_name).load()
 
 
 def load(domain_name=""):
     """Try to load cookies from all supported browsers and return combined cookiejar
     Optionally pass in a domain name to only load cookies from the specified domain
     """
+    logging.info('Loading cookie frim CookieJar')
     cj = http.cookiejar.CookieJar()
     for cookie_fn in [chrome, firefox]:
         try:
             for cookie in cookie_fn(domain_name=domain_name):
                 cj.set_cookie(cookie)
         except BrowserCookieError:
-            pass
+            logging.error('Error in Cookie Loading')

```

Comment:
The errors are not consistent here and in the previous exception handled in `BrowserCookieError(Exception)`. Keep the errors consistent across similar failures.

Reply:
ok, old code got deleted. will go through all today

Reply:
Using BrowserCookieError for handling all exceptions that occur for some reason like file not found or failed to open this etc. So in logging their reason is logged.